approval_polls/templates/approval_polls/poll_update_form.html

```diff
@@ -0,0 +1,69 @@
+{% extends 'base.html' %}
+
+{% block content %}
+    <div class='row-fluid top-buffer'>
+        <div class='col-md-12'>
+            <h3> Edit Poll </h3>
+        </div>
+    </div>
+    <div class='row-fluid top-buffer'>
+        <div class='col-md-12'>
+            <form action="" method="post"> {% csrf_token %}
+                {{ form.non_field_errors }}
+                <div class="fieldWrapper">
+                    <label>Poll Active/Inactive:</label>
+                    {{ form.suspended }}
+                    {% if form.suspended.errors %}
+                        <div class="alert alert-danger" role="alert">
+                            {{ form.suspended.errors }}
+                        </div>
+                    {% endif %}
+                </div>
+                <div class="fieldWrapper">
+                    <label for="{{ form.question.id_for_label }}">Question: </label>{{ form.question }}
+                    {% if form.question.errors %}
+                        <div class="alert alert-danger" role="alert">
+                            {{ form.questions.errors }}
```

Comment:
The error. When I post the form without a question, it shows up a red alert with nothing in it.